src/hotspot/share/memory/virtualspace.hpp

```diff
@@ -61,16 +62,16 @@ class ReservedSpace {
                           size_t page_size, bool special, bool executable);
 
   void initialize(size_t size, size_t alignment, size_t page_size,
-                  char* requested_address, bool executable);
+                  char* requested_address, bool executable, MemTag mem_tag = mtNone);
 
   void reserve(size_t size, size_t alignment, size_t page_size,
-               char* requested_address, bool executable);
+               char* requested_address, bool executable, MemTag mem_tag);
  public:
   // Constructor
   ReservedSpace();
   // Initialize the reserved space with the given size. Depending on the size
   // a suitable page size and alignment will be used.
-  explicit ReservedSpace(size_t size);
+  explicit ReservedSpace(size_t size, MemTag mem_tag);
```

Comment:
```suggestion
  ReservedSpace(size_t size, MemTag mem_tag);
```
No need for `explicit` when you've added a parameter.

Reply:
Fixed.